### ISSUE TITLE
Applet is very small on big monitor with hi-res

### ISSUE THREAD

**Issue by akontsevich** [kotelnik/plasma-applet-weather-widget#63]
Applet and its fonts are very small on big monitor (42") with hi-res (1920x1080) - 52". Seems not designed for Hi/Low DPI monitors. Also applet's content is not resizable so it is not possible to increase (or set) graphic or fonts there. Normal fonts size shown in another weather applet on the left:

![default](https://user-images.githubusercontent.com/3843657/34316036-5eb9a81a-e795-11e7-9330-0270e6546261.png)

Please fix if possible. Thanks!

**Also have question:** how to invoke (show) current weather info like shown on the following plasma-applet-weather-widget screen shot:

![default](https://user-images.githubusercontent.com/3843657/34316045-a8dfbfc4-e795-11e7-8455-d713d25c5515.png)

Thanks!

**Comment by ArchangeGabriel** [comment]
Your version of the widget seems old, I don’t have such issue here. The meteogram is rendered by the widget, and thus the font is the same and I don’t have such a white background either.

Regarding your second question, this is the UI when hovering the icon in the taskbar (while you get the meteogram by clicking on it). Not sure whether you can get it as a widget.

**Comment by akontsevich** [comment]
@ArchangeGabriel, I have the latest - 1.6.10, taken from there: https://store.kde.org/p/998917 Such white diagram rendered for yr.no, for OWM site it is transparent.

>Regarding your second question, this is the UI when hovering the icon in the taskbar

In taskbar? It could be run as and application, not a plasma widget?

**Comment by kotelnik** [comment]
Hi! Regarding the small font in meteogram - it is a downloaded image so yr.no is doing the rendering. But you can let the plasmoid do the rendering (like in case of OWM) by checking "Render meteogram for yr.no" in Appearance settings. If you don't mind it lacks wind and cloud info.

> In taskbar? It could be run as and application, not a plasma widget?

ArchangeGabriel probably meant "panel", not taskbar. I think you cannot have plasmoids as standalone applications. Only for debugging purposes e.g. like this:
```
plasmoidviewer --applet org.kde.weatherWidget
```
...but it is probably not what you want :).

**Comment by akontsevich** [comment]
>But you can let the plasmoid do the rendering (like in case of OWM) by checking "Render meteogram for yr.no" in Appearance settings.

Ok, however "Render meteogram for yr.no" still does not scale fonts or graphic - still very small, and graphic is fixed size.

For the question: panel on screenshot shows wind, pressure, sunrise, sunset info, etc. How to invoke it?

**Comment by kotelnik** [comment]
> still very small, and graphic is fixed size.

You are right that I should fix the widget for different DPIs.

> panel on screenshot shows wind, pressure, sunrise, sunset info, etc. How to invoke it?

This popup with mentioned info can be invoked by just hovering the weather widget by mouse.

**Comment by ArchangeGabriel** [comment]
Yeah, panel/taskbar it’s the same for me. I’m a “traditional” desktop user, with one bar at the bottom for everything (main menu, taskbar, panel…).

**Comment by akontsevich** [comment]
>You are right that I should fix the widget for different DPIs.

Thanks! Good if scale and size storing will be possible.

>This popup with mentioned info can be invoked by just hovering the weather widget by mouse.

Hover over widget? Shows nothing to me on desktop or embedded into taskbar panel.

**Comment by kotelnik** [comment]
> Hover over widget? Shows nothing to me on desktop or embedded into taskbar panel.

That's weird, it should definitely show a popup tooltip. What about standard Digital Clock widget, does this widget show tooltip when hovered by mouse?

**Comment by akontsevich** [comment]
>What about standard Digital Clock widget, does this widget show tooltip when hovered by mouse?

No, it should? Some settings were switched off?

**Comment by kotelnik** [comment]
Yes, Digital Clock should show a tooltip, too. And it cannot be switched off in this widget so I suppose there is some kind of setting for all Qt applications to enable/disable tooltips but I cannot find it.

Maybe you can create a new linux user and test in clean Plasma environment so that default settings is used. Anyway that's what I would do to narrow down the issue.

**Comment by akontsevich** [comment]
I have disabled popup notifications previously because of plasma crash, so popup works now:

![default](https://user-images.githubusercontent.com/3843657/34408364-183d0bee-ebcc-11e7-9a8e-30c6e887ed33.png)

So the only problem for now - small fonts and non re-sizable graphic.
